examples/hpc-c/tvm-hpc/README.md

```diff
@@ -0,0 +1,96 @@
+# Notes
+This build is only tested on python3.11.
+
+# Installing dependencies
+## Installing TVM
+Get sources
+``` sh
+git clone --recursive https://github.com/apache/tvm tvm
```

Comment:
Consider specifying a specific (release?) version. Shallow clone is usually pretty good for large project such as this one if no work on that tree is required (i.e., `--depth=1`)

Reply:
I didn't think about this, good idea!

Reply:
Actually the URL is wrong. I have forked TVM in [https://github.com/vilukissa68/tvm-headsail/tree/dnnl-style](https://github.com/vilukissa68/tvm-headsail/tree/dnnl-style). It should probably be hosted on Soc-Hub-Fi.